javascript/javascript1/week3/homework.js

```diff
@@ -56,3 +56,52 @@ const travelTime = calculateTravelTime(travelInformation);
 console.log(travelTime); // 8 hours and 38 minutes
 
 // SERIES DURATION OF MY LIFE
+
+const seriesDurations = [
+	{
+		title: "House of Cards",
+		days: 2,
+		hours: 12,
+		minutes: 50,
+	},
+	{
+		title: "Ozark",
+		days: 1,
+		hours: 17,
+		minutes: 4,
+	},
+	{
+		title: "Tiger King",
+		days: 0,
+		hours: 9,
+		minutes: 19,
+	},
+];
+
+/*seriesDurations.push({
+	title: "House of Cards",
+	days: 2,
+	hours: 16,
+	minutes: 29,
+});
+*/
+
+function logOutSeriesText(seriesDurations) {
+	// write code here
+	let sumPercentage = 0;
+	for (let i = 0; i < seriesDurations.length; i++) {
+		const timeInMinutes =
+			seriesDurations[i].days * 24 * 60 +
+			seriesDurations[i].hours * 60 +
```

Comment:
can you try to calculate the % of life for one series manually?
 and see what seriesDurations[i].days this shows its value in console.log .
I have an doubt so may be we can discuss this in class

Reply:
ok! can you elaborate? didn't understand...